### ISSUE TITLE
this code doesn't work

### ISSUE THREAD

**Issue by enzomartini** [adelghaenian/Shamsi-Calendar-Rainmeter#1]
you must check the code bro

**Comment by xmha97** [comment]
دقیقا مشکل‌تون کجاست؟
تقویم فارسی به درستی نمایش داده نمی‌شود؟

**Comment by enzomartini** [comment]
> دقیقا مشکل‌تون کجاست؟
> تقویم فارسی به درستی نمایش داده نمی‌شود؟

yes ,,, the app can not fetch exact date ... and the is no separator between H | M | S
